webserver/views/user.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 import webserver
 import db.user
+import ujson
```

Comment:
I'm not saying that you need to use `json` package instead. Just asking for a reason.